pyglmnet/pyglmnet.py

```diff
@@ -448,6 +451,23 @@ class GLM(BaseEstimator):
     verbose : boolean or int
         default: False
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> random_state = 1
+    >>> np.random.seed(random_state)
+    >>> n_samples, n_features = 100, 4
+    >>> X = np.random.randn(n_samples, n_features)
+    >>> y = y = 2.2 * X[:, 0] -1.0 * X[:, 1] + 0.3 * X[:, 3] + 1.0
+    >>> glm = GLM(distr='gaussian', verbose=False, random_state=random_state)
+    >>> glm = glm.fit(X, y)
+    >>> glm.beta0_ # The intercept
+    1.005377890812967
+    >>> glm.beta_ # The coefficients
+    array([ 1.90217526, -0.78781579, -0.        ,  0.03227754])
```

Comment:
sorry my brain is fried at this moment. So I apologize if this is a very dumb question. But can you explain why these are not the same as the coefficients above?

Reply:
See the comment below. I'm not entirely sure - it could be the regularization or the optimization routine. They seem to be quite far off though.

Reply:
hmm ... I wonder if this is related to #275 or #278. Does this also happen when you set the reg to 0?